lua/gitlab/state.lua

```diff
@@ -616,7 +616,17 @@ M.dependencies = {
     endpoint = "/merge_requests",
     key = "merge_requests",
     state = "MERGE_REQUESTS",
-    refresh = false,
+    refresh = true,
+    method = "POST",
+    body = function(argTable)
+      local listArgs = {
+        thishastobehereorelsethiswillbeinterpretedasanarray = {},
```

Comment:
I'm new to lua and go, so any alternative solution here will help @harrisoncramer @jakubbortlik

Reply:
This might work: 

```lua
    body = function(opts)
      return {
        label = opts and opts.label or {},
        notlabel = opts and opts.notlabel or {},
      }
    end,
```